### ISSUE TITLE
Add twisted._threads.test.__init__.py to dist3.py so that more _threads tests run on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8540]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8540|
|Type|enhancement|
|Created|2016-06-30 01:57:47Z|
|Branch|https://github.com/twisted/twisted/tree/8540-rodrigc-thread|

In previous porting work, the _threads tests were added to
dist3.py, but the __init__.py file for the tests was omitted
(maybe by accident).  This prevents twisted._threads.test from
running under Python 3.

twisted/_threads/test/__init__.py needs to be added to dist3.py

<details><summary>Searchable metadata</summary>

```
trac-id__8540 8540
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8540_rodrigc_thread 8540-rodrigc-thread
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467251867433278 1467251867433278
changetime__1467272413121615 1467272413121615
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/273

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @rodrigc|
|-|-|

LGTM. Merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 33b265475aac728163dfb5be6c8d84e205f9a21c
```
#!CommitTicketReference repository="" revision="33b265475aac728163dfb5be6c8d84e205f9a21c"
Merge 8540-rodrigc-thread: Add twisted/_threads/test/__init__.py to dist3.py to run more threads tests

Author: rodrigc
Reviewer: Lukasa
Fixes: #8540
```
